source/about/editions-and-offerings.md

```diff
@@ -0,0 +1,172 @@
+- **Desktop Application** for Windows, MacOS, and Linux (Ubuntu/Debian, CentOS/RHEL, Generic Linux) including support for [Desktop MSI and Group Policies (Beta)](https://docs.mattermost.com/install/desktop-msi-installer-and-group-policy-install.html), [silent installation](https://docs.mattermost.com/install/desktop-msi-installer-and-group-policy-install.html#silent-installation-guide), and [managed resources](https://docs.mattermost.com/install/desktop-app-managed-resources.html). 
+- [**Integrations**](https://docs.mattermost.com/about/integrations.html) including connected workflows with common developer tools, including Jira, Confluence, GitHub, GitLab, CircleCI, Zoom, and Jitsi, as well as an extensible support for [bots and plug-ins](https://mattermost.com/marketplace/#publicApps) as well as [webhooks, APIs, and drivers](https://developers.mattermost.com/integrate/other-integrations). 
+- **Multi-language support** including English (U.S. and Australian), Bulgarian, Chinese (Simplified and Traditional), Dutch, French, German, Italian, Japanese, Korean, Persian, Polish, Portuguese (Brazil), Romanian, Russian, Spanish, Swedish, Turkish, and Ukrainian.
+
+System management and customization features include:  
+- [Tools for Custom Branding](https://docs.mattermost.com/configure/custom-branding-tools.html) including customization of login page. 
+- [Custom Themes and Colors](https://docs.mattermost.com/messaging/customizing-theme-colors.html) including ability to import templates. 
+- [Multi-factor authentication](https://docs.mattermost.com/onboard/multi-factor-authentication.html) via Google Authenticator in addition to username/password and [password policy enforcement](https://docs.mattermost.com/configure/configuration-settings.html#password). 
+- (Self-hosted Only) [Command Line Tool](https://docs.mattermost.com/manage/mmctl-command-line-tool.html) for configuration and [server upgrade](https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html). 
+
+Support entitlements from Mattermost, Inc: 
+- [Knowledge base and troubleshooting documentation](https://support.mattermost.com/hc/en-us) is available for common techincal issues. 
+- [Peer-to-Peer community support](https://mattermost.com/support/) via on-line forums. 
+
+Service entitlements from Mattermost, Inc: 
+- (Cloud Only) [SOC 2 Type 1 certification](https://mattermost.com/security/) with plans to add SOC 2 Type 2 in 2023. 
+
+
+### Mattermost Professional 
+
+Mattermost Professional is a paid subscription offering advanced configuration for software organizations from 25 to 100 users. 
```

Comment:
Recommend rewording or clarifying that "25 to 100 users" is just an ideal or guide.